packages/web-components/fast-foundation/src/checkbox/checkbox.template.ts

```diff
@@ -27,10 +27,10 @@ export const checkboxTemplate: (
             x.checked ? "checked" : ""} ${x => (x.indeterminate ? "indeterminate" : "")}"
     >
         <div part="control" class="control">
-            <slot name="checked-indicator">
+            <slot name="checked-indicator" aria-hidden="true">
```

Comment:
I haven't seen aria attributes used directly on a slot element. Are we sure that has the desired effect?

Reply:
I wondered that too. How do I get the element to show up in AT at all? Then I can test if it works on slots. I figured it did because they are just an element and the spec says it applies to everything on and below the element. Slots are somewhat special though, so I could see some obscure reason it doesn't work. Our other patterns seems to be to place this on a span around the slot.

Reply:
The contents here are set in the compose call in fast-components. The default content for foundation is empty.

Reply:
Thanks Chris for confirming. I removed the aria-hidden tag for any child of an element with [applicable roles](https://w3c.github.io/aria-practices/#children_presentational).